tests/models/multiple_choice_qa/tuple_inference_model_test.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=no-self-use,invalid-name
+from unittest import TestCase
+import os
+import shutil
+
+from numpy.testing.utils import assert_allclose
+
+from deep_qa.models.multiple_choice_qa.tuple_inference import TupleInferenceModel
+from ...common.constants import TEST_DIR
+from ...common.models import get_model, write_tuple_inference_files
+
+class TestTupleInferenceModel(TestCase):
+    def setUp(self):
+        os.makedirs(TEST_DIR, exist_ok=True)
+        write_tuple_inference_files()
+
+    def tearDown(self):
+        shutil.rmtree(TEST_DIR)
+
+    def test_train_does_not_crash(self):
+        args = {"tuple_match": {"num_hidden_layers": 1,
+                                "hidden_layer_width": 4,
+                                "hidden_layer_activation": "tanh"},
+                "num_question_tuples": 5,
+                "num_background_tuples": 5,
+                "num_tuple_slots": 4,
+                "word_sequence_length": 10,
+                "num_answer_options": 4}
+        solver = get_model(TupleInferenceModel, args)
+        solver.train()
+
+    def test_model_trains_and_loads_correctly(self):
+        model = get_model(TupleInferenceModel, {'save_models': True})
+        model.train()
+
+        loaded_model = get_model(TupleInferenceModel)
```

Comment:
I would combine these two tests, so that we don't run the training twice in the tests (it takes at least several seconds, sometimes as much as 30-40).  Just add from here down into the other test, and use the name from this one.